docs/source/troubleshooting/troubleshooting-getting-started.adoc

```diff
@@ -138,3 +138,14 @@ Downloading https://github.com/minishift/minishift/releases/download/v1.2.0/mini
  65 B / 65 B [===================================================================================================================================================] 100.00% 0s
 Update failed: open /usr/bin/.minishift.new: permission denied
 ----
+
+[[safari-openshift-web-console-issue]]
+== Safari browser issue with OpenShift web console
```

Comment:
We can be more specific and say that the OpenShift web console does not work with some versions of Safari here. "Safari browser issue" is very vague.

Reply:
Agree